src/search/search-command.js

```diff
@@ -25,11 +25,17 @@ function runSearchAndSendResults(command, bot, message, query, attachmentTitle)
       });
     })
     .catch(err => {
-      command.handleError(bot, message, err, err => {
-        bot.reply(message, 'Unknown error occurred while performing your search.\n' +
-          'Please try again later or contact support.');
-        logger.error(`Unknown error occurred while performing user search: [${JSON.stringify(query)}]`, err);
-      });
+      if (err.code === 429) {
```

Comment:
move check to https://github.com/logzio/slack-integration/blob/master/src/core/client/http-client.js#L73

Reply:
Done